syllabus/module/2.3-slides.md

```diff
@@ -0,0 +1,128 @@
+---
```

Comment:
it must be "Smart pointers & Macros" and you need to cover the basic concepts of Macros too